app/cdap/components/PipelineTriggers/store/PipelineTriggersActionCreator.ts

```diff
@@ -636,3 +636,11 @@ function _transformSchedule(existingSchedules: ISchedule[]) {
 
   return [...transformedSchedules, ...enabledGroupSchedule];
 }
+
+/**
+ * Method to transform the single trigger to OR group trigger.
+ * @param link - the link to route to.
+ */
+export const handlePipelineLinkClick = (link: string) => {
```

Comment:
I'd move this to `services/helpers` or the `shared` directory as it's not really creating an action in the store (no calls to `PipelineTriggersStore` or state changes). It is also being used by more than one component. You could also use a generic name such as `openLinkInNewTab` or something similar.

Reply:
Good idea, will update